test/IISExpress.FunctionalTests/RequiresIISAttribute.cs

```diff
@@ -10,15 +11,24 @@ namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
     [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class | AttributeTargets.Method)]
     public sealed class RequiresIISAttribute : Attribute, ITestCondition
     {
-        public bool IsMet => IISExpressAncmSchema.SupportsInProcessHosting;
+        public bool IsMet { get; set; } = IISExpressAncmSchema.SupportsInProcessHosting;
 
-        public string SkipReason => IISExpressAncmSchema.SkipReason;
+        public string SkipReason { get; set; } = IISExpressAncmSchema.SkipReason;
 
         public RequiresIISAttribute() { }
 
         public RequiresIISAttribute(IISCapability capabilities)
         {
-            // IISCapabilities aren't pretinent to IISExpress
+            if (capabilities.HasFlag(IISCapability.Admin))
```

Comment:
It's not an IIS capability.

Reply:
I would have to create a new enum otherwise.

Reply:
You can create a new attribute.